examples/reverse_words/main.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""Learm to reverse words of natural text."""
```

Comment:
Also, I think we talk about *natural language*, but *natural text* isn't really a thing, so I would just say *Learn to reverse the words in a text.*